test/Test/Indigo/Contracts/HomebaseLite/PropertyTests.hs

```diff
@@ -93,8 +93,8 @@ hprop_main = withTests 500 $ property $ do
       verifyStorage
         :: forall caps m'. MonadEmulated caps m'
         => ContractHandle Parameter Storage ()
-        -> (forall m. MonadCleveland caps m => Storage -> m ())
-        -> (forall m. MonadCleveland caps m => m ())
+        -> (Storage -> m' ())
+        -> m' ()
```

Comment:
This was actually intentional -- otherwise, verification functions have access to the full `MonadEmulated` context. Which is I guess is not a big deal.

Reply:
Reverted